snapcraft/main.py

```diff
@@ -23,11 +23,13 @@
   snapcraft [options]
 
 Options:
-  -h --help        show this help message and exit
-  -v --version     show program version and exit
-  -V --verbose     print additional information about command execution
-  -d --debug       print debug information while executing (including
-                   backtraces)
+  -h --help              show this help message and exit
+  -v --version           show program version and exit
+  -V --verbose           print additional information about command execution
+  -d --debug             print debug information while executing (including
+                         backtraces)
+  --no-parallel-build    use only a single build job per part (defaults to
+                         number of CPUs)
```

Comment:
Hmm, how about "the default number of jobs per part is equal to the number of CPUs"?